src/main/java/seedu/duke/helper/Parser.java

```diff
@@ -19,4 +22,71 @@
     public static String[] commandParser(String userInput) {
```

Comment:
is it parserCommand instead of commandParser since it is a noun? Or does it need to be changed to sound like a verb such as parseNewInput?